api/envoy/config/filter/http/adaptive_concurrency/v3alpha/adaptive_concurrency.proto

```diff
@@ -49,6 +49,13 @@ message GradientControllerConfig {
     // The number of requests to aggregate/sample during the minRTT recalculation window before
     // updating. Defaults to 50.
     google.protobuf.UInt32Value request_count = 2 [(validate.rules).uint32.gt = 0];
+
+    // Randomized time delta that will be introduced to the start of the minRTT calculation window.
```

Comment:
Please make all your changes in v2 still. Everything to v3 will be auto synced. Also keep using v2 below.